arangod/Graph/TraverserDocumentCache.cpp

```diff
@@ -149,8 +149,8 @@ void TraverserDocumentCache::insertDocument(
           cache::CachedValue::construct(key, keySize, resVal, resValSize));
 
       if (value) {
-        bool success = _cache->insert(value.get());
-        if (!success) {
+        auto result = _cache->insert(value.get());
+        if (!result.ok()) {
           LOG_TOPIC(DEBUG, Logger::GRAPHS) << "Insert document into cache failed";
```

Comment:
The intention was to see what is happening with Graph Searches of user complain it is to slow. Then in Debug Mode we can see if caches are filled properly or not.